### ISSUE TITLE
Update 'Skip to Content' Link

### ISSUE THREAD

**Issue by joncameron** [avalonmediasystem/avalon#5440]
# Description
The skip to content link sits at the top of the page as the first focusable link for easily moving focus to the content area—for Avalon, the media player. Current line:
https://github.iu.edu/AvalonIU/Avalon/blob/7e3a0a36b915d0f3ced981c926a3edce02f41cc7/app/views/layouts/avalon.html.erb#L43

This should be updated to link to the Ramp media player, rather than `#content`. See https://www.w3.org/TR/WCAG20-TECHS/G1.html.

# Done Looks Like
- [x] Skip to main content link goes to the appropriate page element (#iiif-media-player?)
- [x] This link is the first focusable control on the Web page.
- [x] The description of the link communicates that it links to the main content.
- [x] The link is visible when it has keyboard focus.
- [x] Activating the link moves the focus to the main content.
- [x] After activating the link, the keyboard focus has moved to the main content.
